lua/nvim-tree/api.lua

```diff
@@ -186,7 +225,25 @@ Api.tree.search_node = wrap(actions.finders.search_node.fn)
 ---@field keep_buffers boolean|nil default false
 
 Api.tree.collapse_all = wrap(actions.tree.modifiers.collapse.all)
+
+---@class ApiTreeExpandAllOpts
+---@field descend_until (fun(expansion_count: integer, node: Node): boolean)|nil
```

Comment:
```suggestion
---@field expand_until (fun(expansion_count: integer, node: Node): boolean)|nil
```

That would match the name of the api methods: expand